### ISSUE TITLE
Highlight VCF or variants table records alongside metadata

### ISSUE THREAD

**Issue by peterk87** [peterk87/shiptv#12]

**Comment by nhhaidee** [comment]
@peterk87 I ran tests for phylogenetic analysis (without filtering GISAID) and am thinking about adding supports for SNPs visualization? This is what's you are mentioning in this enhancement?
[phylogentic_tree_snps.pdf](https://github.com/peterk87/shiptv/files/6727500/phylogentic_tree_snps.pdf)

**Comment by peterk87** [comment]
@nhhaidee I don't think this enhancement is necessary if SNP or mutation information is added to the metadata table (e.g. Nextclade AA mutations merged with Pangolin and GISAID metadata in the [scovtree pipeline](https://github.com/nhhaidee/scovtree)). Support for visualizing variants from VCF files would be nice, but not essential as part of shiptv at this time.
